synthtool/languages/node.py

```diff
@@ -317,14 +317,6 @@ def owlbot_main(
             s_copy([library], excludes=staging_excludes)
         # The staging directory should never be merged into the main branch.
         shutil.rmtree(staging)
-    else:
```

Comment:
Many pull requests for repos don't have staging directories at all, so I think we still need this code.